src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -271,8 +271,7 @@ private CodeAction CreateAction(State state, bool allOccurrences, bool isConstan
             var declaringType = semanticModel.GetEnclosingNamedType(expression.SpanStart, cancellationToken);
             var reservedNames = declaringType.GetMembers().Select(m => m.Name);
 
-            return syntaxFacts.ToIdentifierToken(
```

Comment:
`ISyntaxFactsService.ToIdentifierToken` was only used in `ISemanticFactsService.GenerateUniqueName` implementation, and was really not a syntax facts helper anyways. `ISyntaxFactsService.ToIdentifierToken` has now been removed.